pkg/generators/openapi.go

```diff
@@ -603,7 +603,8 @@ func (g openAPITypeWriter) generateDescription(CommentLines []string) {
 		}
 	}
 
-	postDoc := strings.TrimRight(buffer.String(), "\n")
+	postDoc := strings.TrimLeft(buffer.String(), "\n")
```

Comment:
Note: After we get go 1.18 trims will be alloc-free and faster. ref: https://tip.golang.org/doc/go1.18